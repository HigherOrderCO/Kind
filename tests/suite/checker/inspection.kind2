Main : U60
Main =
  ?