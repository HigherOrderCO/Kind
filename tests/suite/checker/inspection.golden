[4mInspection.[0m
- Goal: U60
[4mOn '{{#F0F#}}':[0m
{{#R0:20:21R#}}

